### PR TITLE
SC-30(4) text update

### PR DESCRIPTION
**Pull request by thamill**
updated text for customer responsibility

### CODE REVIEW

**Comment by shawndwells** [comment]
Also, the files under ``/build/`` should be dropped from pull requests. Those are dynamically generated.

**Comment by thamill** [comment]
The text of the control is that it relies on the organization to define the scope of system components for assessment. 
Wouldn't accepting 'partial' or 'complete' as a status cause Red Hat to accept responsibility of satisfaction for all clients that choose to place OCP and COS in scope?

File: coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 1]
```diff
@@ -1220,10 +1220,12 @@
...
-        To assist with this organizational control, documentation is being
-        planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-515
+        The customer is solely responsible for implementing a strategy to mislead
+        adversarial actors with regard to the organizations securite state or posture.
+        This is a control satisfied by organizational policies and procedures.
+        Deploying misleading information through CoreOS ports can be
+        configured using the OpenShift Machine Config Operator, including presented
+        SSH banners.
```
Comment (redhatrises) [inline coreos-4/policies/SC-Systems_and_Communications_Protection/component.yaml:1228]:
```suggestion
        configured using the OpenShift Machine Config Operator.
```

File: openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 2]
```diff
@@ -1241,9 +1241,17 @@
...
-
+        The customer is solely responsible for implementing a strategy to mislead
+        adversarial actors with regard to the organizations securite state or posture.
+        This is an organizational policy and procedure control that OpenShift configuration
+        cannot satisfy, although use of the product could be part of the implementation strategy.
+        As part of a misleading strategy, the OpenShift Web Console Operator that supplies the
+        web console interface can be configured to supply different or reduced sets of HTTP
+        headers.
+        Console Operator documentation:
+        https://github.com/openshift/console-operator
```
Comment (redhatrises) [inline openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml:1252]: Feels like this is a copy/paste from another control and doesn't really address "misleading information"

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**